packages/lib/sync/SyncServiceManager.ts

```diff
@@ -0,0 +1,72 @@
+import logger from "@calcom/lib/logger";
+
+import { ConsoleUserInfoType, WebUserInfoType } from "./ISyncService";
+import services from "./services";
+
+const log = logger.getChildLogger({ prefix: [`[[SyncServiceManager] `] });
+
+export const createConsoleUser = async (user: ConsoleUserInfoType | null | undefined) => {
+  if (user) {
+    log.debug("createConsoleUser", { user });
+    try {
```

Comment:
The idea is that Sync Services may fail and shouldn't block any workflow in the app as it's completely secondary to its purpose.